src/library/autodj/autodjprocessor.cpp

```diff
@@ -566,6 +559,8 @@ AutoDJProcessor::AutoDJError AutoDJProcessor::toggleAutoDJ(bool enable) {
                 setCrossfader(1.0);
             }
         }
+        m_crossfaderStartCenter = m_pConfig->getValue<bool>(ConfigKey(kConfigKey,
+                QStringLiteral("center_xfader_when_starting_xfade")));
```

Comment:
please set the default here
```suggestion
        m_crossfaderStartCenter = m_pConfig->getValue<bool>(ConfigKey(kConfigKey,
                QStringLiteral("center_xfader_when_starting_xfade")),
                false);
```